### ISSUE TITLE
How to use DM with ResNet18BN 

### ISSUE THREAD

**Issue by chenxiang3luo** [VICO-UoE/DatasetCondensation#32]
Hi, I want to use DM with ResNet18BN and test in the ResNet18BN, but the performance is poor e.g. 12%(worse than random selection which is 14.6% in your report) under the setting where dataset=cifar100, ipc=10. I only replace the --model  parameter. I don't know if that's normal. Do I need to change other hyperparameters?. Thanks very much for your replay.

**Comment by PatrickZH** [comment]
Hi, please refer to the closed issues on this question.  
https://github.com/VICO-UoE/DatasetCondensation/issues/30
https://github.com/VICO-UoE/DatasetCondensation/issues/11
https://github.com/VICO-UoE/DatasetCondensation/issues/8

**Comment by chenxiang3luo** [comment]
Thank you very much. I will try it with smaller learning rate.
